unittest/IlmCtl/CMakeLists.txt

```diff
@@ -57,7 +56,14 @@ file(
         testVarying.ctl
         testVaryingLookup.ctl
         testVaryingReturn.ctl
-        testVSArrays.ctl
-    DESTINATION
-        ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}
-)
+        testVSArrays.ctl)
+        
+if (MSVC)
+  file(COPY ${ILM_CTL_TEST_FILES} DESTINATION ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/Debug)
+  file(COPY ${ILM_CTL_TEST_FILES} DESTINATION ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/Release)
```

Comment:
As discussed we  need to check if we are able to build successfully in x86 as well